### ISSUE TITLE
ArchLinux need qt5-tools、qt5-quickcontrols、qt5-graphicaleffects

### ISSUE THREAD

**Issue by VliuPro** [xiangzhai/qwx#34]
```
-- Debug build
CMake Error at /usr/lib64/cmake/Qt5/Qt5Config.cmake:26 (find_package):
  Could not find a package configuration file provided by "Qt5LinguistTools"
  with any of the following names:

    Qt5LinguistToolsConfig.cmake
    qt5linguisttools-config.cmake

  Add the installation prefix of "Qt5LinguistTools" to CMAKE_PREFIX_PATH or
  set "Qt5LinguistTools_DIR" to a directory containing one of the above
  files.  If "Qt5LinguistTools" provides a separate development package or
  SDK, be sure it has been installed.
Call Stack (most recent call first):
  CMakeLists.txt:45 (find_package)


-- Configuring incomplete, errors occurred!
```

install qt5-tools

```
sudo pacman -S qt5-tools
```

***

```
QQmlApplicationEngine failed to load component
qrc:/qml/main.qml:4 module "QtQuick.Controls" is not installed
```

install qt5-quickcontrols

```
sudo pacman -S qt5-quickcontrols
```

打开聊天界面报错，缺少QtGraphicalEffects

```
sudo pacman -S qt5-graphicaleffects
```

暂时就这么多0.0

**Comment by xiangzhai** [comment]
O(∩_∩)O谢谢

**Comment by MingcongBai** [comment]
Might try using commits with references to the issue number for better tracking.

**Comment by xiangzhai** [comment]
> Might try using commits with references to the issue number for better tracking.

Correct ;-) https://github.com/xiangzhai/qwx/commit/ed4174886766baa990dee8793f0a92aa50bec2e7

**Comment by MingcongBai** [comment]
What you could do was something like `README: add missing dep for Arch Linux; fix #34`, and that commit will automatically close this issue once pushed.

**Comment by xiangzhai** [comment]
oh, it just like KDEBUG and REVIEW hook ;-)
